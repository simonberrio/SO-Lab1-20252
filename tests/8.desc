reverse: reverses a large file with many lines
