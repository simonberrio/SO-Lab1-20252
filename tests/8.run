./reverse tests/8.in tests-out/8.out
