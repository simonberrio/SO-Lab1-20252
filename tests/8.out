line7
line6
line5
line4
line3
line2
line1
